### PR TITLE
ci: add java package

### PR DESCRIPTION
**Pull request by sboukhetta**
Add java package release

### CODE REVIEW

File: packages/java/pom.xml [hunk 1]
```diff
@@ -4,11 +4,40 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.aneo</groupId>
+  <groupId>fr.aneo</groupId>
```
Comment (ngruelaneo) [inline packages/java/pom.xml:7]: perhaps `eu.aneo` should be better. aneo.fr is only a redirection I think.

File: packages/java/pom.xml [hunk 2]
```diff
@@ -4,11 +4,40 @@
...
   <artifactId>armonik-java</artifactId>
   <description>GRPC java binding for the Armonik orchestrator API</description>
   <version>0.1.0</version>
 
+	<name>armonik-java</name>
+	<url>https://github.com/aneoconsulting/ArmoniK.Api</url>
+
+	<developers>
```
Comment (ngruelaneo) [inline packages/java/pom.xml:15]: I think that we should add Hassan who start it even if he is not anymore with aneo.

File: packages/java/pom.xml (hunk 1 above)
Comment (jdongo-aneo) [reply packages/java/pom.xml:7]: the namespace has been created with the fr.aneo after discussing with Maxime

File: packages/java/pom.xml (hunk 2 above)
Comment (sboukhetta) [reply packages/java/pom.xml:15]: Yes, I will add him.

**Review by dbrasseur-aneo — APPROVED** [review]

**Pull request merged**